marketplace-service/src/main/java/com/axonivy/market/github/service/impl/GHAxonIvyProductRepoServiceImpl.java

```diff
@@ -119,6 +129,36 @@ public void extractReadMeFileFromContents(Product product, List<GHContent> conte
     }
   }
 
+  @Override
+  public void updateSetupPartForProductModuleContent(Product product,
+      Map<String, Map<String, String>> moduleContents, String tag) throws IOException {
+    if (!NonStandardProduct.isMsGraphProduct(product.getId())) {
+      return;
+    }
+
+    GHRepository ghRepository = gitHubService.getRepository(product.getRepositoryName());
+    List<GHContent> contents = ghRepository.getDirectoryContent(MS_GRAPH_PRODUCT_DIRECTORY, tag);
+
+    GHContent setupFile = contents.stream().filter(GHContent::isFile)
+        .filter(content -> content.getName().equalsIgnoreCase(SETUP_FILE))
+        .findFirst().orElse(null);
+
+    if (ObjectUtils.isNotEmpty(setupFile)) {
+      String setupContent = new String(setupFile.read().readAllBytes());
+      if (ProductContentUtils.hasImageDirectives(setupContent)) {
+        List<GHContent> setupImagesFolder =
+            contents.stream().filter(content -> content.getName().equals(MG_GRAPH_IMAGES_FOR_SETUP_FILE)).toList();
+        setupContent = updateImagesWithDownloadUrl(product, setupImagesFolder, setupContent);
+      }
+
+      if (setupContent.contains(ReadmeConstants.SETUP_PART)) {
+        List<String> extractSetupContent = List.of(setupContent.split(ReadmeConstants.SETUP_PART));
+        setupContent = ProductContentUtils.removeFirstLine(extractSetupContent.get(1));
+      }
+      ProductContentUtils.addLocaleContent(moduleContents, SETUP, setupContent, Language.EN.getValue());
+    }
```

Comment:
this brakes the build contract of market products; they are designed as Maven repository that can define it's description via Maven jar.

now you leak it form github, therefore breaking all advanced products that made use of the maven ecosystem.